aptos-move/block-executor/src/executor.rs

```diff
@@ -102,7 +102,7 @@ where
         executor: &E,
         base_view: &S,
         latest_view: ParallelState<T, X>,
-    ) -> ::std::result::Result<bool, PanicOr<IntentionalFallbackToSequential>> {
+    ) -> Result<bool, PanicOr<IntentionalFallbackToSequential>> {
```

Comment:
nit: I'd leave ::std::result (Result is too generic of a term, good to know where its coming from). Unless this is a different "Result"

Reply:
Isn't it better to qualify other results, e.g. anyhow::Result, etc. but keep the standard result unqualified? Don't know what is the best practice

Reply:
yeah, here I disagree:
- convention across the codebase is to not qualify this, Result is a super standard part of language (like Option)
- std is standard, we never qualify anything std, we qualify other things, e.g. anyhow::
- that's why renaming our specialized Result to explain what it is.